config.go

```diff
@@ -200,6 +205,7 @@ func DefaultLANConfig() *Config {
 		ProbeTimeout:            500 * time.Millisecond, // Reasonable RTT time for LAN
 		ProbeInterval:           1 * time.Second,        // Failure check every second
 		DisableTcpPings:         false,                  // TCP pings are safe, even with mixed versions
+		AwarenessMaxMultiplier:  7,                      // Probe interval backs off to (7 + 1) seconds
```

Comment:
Is the maximum 7 seconds (e.g. its multiplied by 7) or its scaled by 7 (max is 8). Don't mean to be pedantic but its just unclear to me